Resources/Private/Templates/DashboardWidgets/IframeWidget.html

```diff
@@ -0,0 +1,3 @@
+<f:if condition="{url}">
+  <iframe class="iframe-widget" src="{url}" width="100%" height="100%" scrolling="{scrolling}"></iframe>
```

Comment:
@MattiasNilsson  should I remove the scrolling option? I'm not sure it is needed.